test/core/utils/is-xhtml.js

```diff
@@ -0,0 +1,21 @@
+describe('axe.utils.isXHTML', function () {
+	'use strict';
+
+	it('should be a function', function () {
+		assert.isFunction(axe.utils.isXHTML);
+	});
+
+	it('should return true on any document that is XHTML', function () {
+		var doc = document.implementation.createDocument('http://www.w3.org/1999/xhtml', 'html', null);
```

Comment:
I had no idea this interface existed. 

![the-more-you-know](https://user-images.githubusercontent.com/1045233/32028615-6118a800-b9a4-11e7-88d9-f92ace2bce57.png)